man/systemd.link.xml

```diff
@@ -612,6 +612,20 @@
           </para>
         </listitem>
       </varlistentry>
+      <varlistentry>
+        <term><varname>ReceiveChecksumOffload=</varname></term>
+        <listitem>
+          <para>Takes a boolean. If set to true, the receive check summing (Rx) is enabled.
```

Comment:
maybe "the receive check summing" → "hardware offload for checksumming of ingress network packets"?